protractor/e2e/integration/ng2.spec.ts

```diff
@@ -58,5 +58,13 @@ describe('Main page test suite', () => {
         await secondPage.infoSubmitButton.click();
     });
 
+    it('Check blog page',async () => {
+        await secondPage.blogButton.click();
+        expect(await secondPage.getCurrentUrl()).toContain(secondPage.blogLink);
+        expect(await secondPage.postsList.count()).toEqual(20);
+        expect(secondPage.isElementDisplayed(await secondPage.navigationBar)).toBeTruthy();
+        await secondPage.linkTosSecondPage.click();
+        expect(await secondPage.postsList.count()).toBeTruthy()
```

Comment:
Then second page opens and there is as minimum 1 article (page is not empty) - this is task
You check only count of posts to be true. In this case, will be more clear:
expect(await browser.getCurrentUrl()).toContain('page/2/');
        expect(await $$('article header').count()).not.toBeNull();